dependencies.props

```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+1.0.1-prerelease-01401-04<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Ah, it's because it's downgrading `Microsoft.xunit.netcore.extensions`, and [UpdateDependenciesAndSubmitPullRequest.cs#L82-L89](https://github.com/dotnet/buildtools/blob/f89d703112923c2e33d0b3b6135304083978fe1b/src/Microsoft.DotNet.Build.Tasks/VersionTools/UpdateDependenciesAndSubmitPullRequest.cs#L82-L89) picks up the `Version` metadata of the `StaticDependency` and misinterprets it. The XML node it tries to update to "current" doesn't exist, so it sticks the version at the top.

This won't repro once `Microsoft.xunit.netcore.extensions` is updated and kept in sync. Filed https://github.com/dotnet/buildtools/issues/1374 to track a fix.